website/docs/advanced-concepts/multiple-environments.md

```diff
@@ -0,0 +1,95 @@
+
+# Multiple Environments
+
+Environments are container entities within a workspace that let you create different updates to different versions of your content types while still keeping them all in isolation. This process is often utilized when developing, identifying errors, or introducing new features without any concerns about how it might influence the current production.
+
+:::info
+Multiple Environments are only available on [Business Edition](https://www.appsmith.com/pricing). 
+:::
+
+Software developers can run numerous tests in these environments to filter out bugs and raise the application's overall quality. Different testing settings can be used to examine different aspects of a software application, including its performance, user interface, security, and more.
+
+## Environments on Appsmith
+There are two different environments that are available on Appsmith:
```

Comment:
Before mentioning, add the line shared in the first draft

> Appsmith provides a way for you to define different environments for datasources. It allows you to switch between different environments as you develop your application in the Edit mode. By switching environments, you can apply the datasource configurations defined in that particular environment.

Reply:
done